app/bundles/LeadBundle/Translations/en_US/messages.ini

```
@@ -263,6 +263,8 @@ mautic.lead.lead.events.field_value="Contact field value"
 mautic.lead.lead.events.field_value_descr="Condition based on a contact field value."
 mautic.lead.lead.events.device="Contact device"
 mautic.lead.lead.events.device_descr="Condition based on a contact device."
+mautic.lead.lead.events.pageHit="Visited page"
+mautic.lead.lead.events.pageHit_descr="Condition based on a contact visited pages"
```

It might be useful to highlight that this is not about the page hit that happened just now or recently but about the all page hits the contact has ever made. Something like:?
```suggestion
mautic.lead.lead.events.pageHit_descr="Condition based on all the pages the contact has visited in the past"
```